scripts/background.js

```diff
@@ -0,0 +1,13 @@
+chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
```

Comment:
This code actually isn't doing anything. I'm leaving it because I think I'll need a service worker in the future.